examples/bip32.rs

```diff
@@ -36,7 +37,8 @@ fn main() {
     let seed = wif.to_bytes();
 
     // we need secp256k1 context for key derivation
-    let secp = Secp256k1::new();
+    let mut buf: Vec<AlignedType> = Vec::with_capacity(Secp256k1::preallocate_size());
+    let secp = Secp256k1::preallocated_new(buf.as_mut_slice()).unwrap();
```

Comment:
Wait, how does this work? `with_capacity` doesn't actually give the vec any size, so `buf` should be 0-length and `preallocated_new` should fail, no? Do the examples get run in CI?

Reply:
it wasn't running in CI.  fixed, and added to CI.